.circleci/test-deploy.yml

```
@@ -1,32 +1,29 @@
 version: 2.1
 orbs:
-  aws-elastic-beanstalk: circleci/aws-elastic-beanstalk@dev:<<pipeline.git.revision>>
   aws-cli: circleci/aws-cli@3.1
```

We still leave the orb defined here to illustrate its injection. It is not required but a good to have.

```suggestion
  aws-elastic-beanstalk: {}
  aws-cli: circleci/aws-cli@3.1
```